packages/form-js-viewer/src/index.js

```diff
@@ -1,46 +1,40 @@
-import { FormCore } from './core';
-import { FormRenderer } from './rendering';
-
-const schemaVersion = 1;
-
-export * from './core';
-
-export * from './rendering';
+import Form from './Form';
 
+export * from './Form';
+export * from './render';
 export * from './util';
 
+const schemaVersion = 1;
+
 export { schemaVersion };
 
 /**
- * @typedef { { container: Element; schema: any; data: any; properties?: any } } FormViewerOptions
+ * @typedef { { [x: string]: any } } Data

```

Comment:
We do export `FormEditor` with `form-js-editor`, however we do not export `FormViewer`. Is this intentional?

Reply:
Done (cf. https://github.com/bpmn-io/form-js/pull/75/commits/7c55ac1cec126d0d177d9c201d65715334cf0156).